example/main.go

```diff
@@ -112,7 +112,7 @@ func main() {
 			Client:             cmclient.NewForConfigOrDie(restConfig),
 			MetadataReader:     store,
 			Clock:              clock.RealClock{},
-			Log:                log,
+			Log:                &log,
```

Comment:
logr.Logger was an interface but now is a struct, so I've made it a pointer in the Options, so that we can tell when it has been set.